platform/admin/core/admin-core-insecuredefaults/src/main/java/org/codice/ddf/admin/insecure/defaults/service/InsecureDefaultsCollector.java

```diff
@@ -244,4 +273,37 @@ private String getTruststorePassword() {
   List<Validator> getValidators() {
     return validators;
   }
+
+  SystemNotice createUsersPropertiesNotice() {
+    UserPropertiesDeletionScheduler scheduler = new UserPropertiesDeletionScheduler();
+    Date firstInstall = scheduler.installationDate();
+    Set<String> details = new HashSet<>();
+    String title;
+
+    if (firstInstall != null) {
+      Date timestamp = new Date(firstInstall.getTime() + TimeUnit.DAYS.toMillis(3));
+
+      title =
+          String.format(
+              "USERS.PROPERTIES FILE FOUND! Follow the instructions below before %s to prevent the system from locking down.",
+              timestamp.toString());
+      details.add(
+          String.format(
+              "A user.properties file was found at [%s]. It will be automatically deleted on %s. After the deletion of this file you will not be able to log back in to the system. Please follow the hardening guides in the documentation under Removing Default Users to avoid the system lock down.",
```

Comment:
✏️ Printing out the path to the properties file leaks file system information and should probably be avoided. While the admin should already know this path, it would be best not to leak it here.